docs/evaluation.md

```diff
@@ -90,6 +90,6 @@ It can be called like so
 ```bash
 any-agent-evaluate \
     --evaluation_case_path "docs/examples/evaluation_case.yaml" \
-    --trace_path "tests/unit/evaluation/sample_traces/OPENAI.json" \
```

Comment:
This move makes the dir location cleaner, but will help us out more too if we go with adding some snapshot tests #282